src/formatDate.js

```diff
@@ -49,8 +49,43 @@
  * @returns {string}
  */
 
+const YY = 'YY';
+const YYYY = 'YYYY';
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const objDate = {};
```

Comment:
is it necessary to use Object? 


asking cause i had the same solution, but mentor didn't approve it until i remove an object
and use ONLY arrays to resolve a problem

Reply:
@Anton-Kuchmasov the option using an object is also suitable, it will look better simply through an array